### DISCUSSION TITLE
Speedup suggestion...  One line (hah, not)..

### DISCUSSION THREAD

**Discussion by KC5VDJ** [kej715/DtCyber#138] [General]
Changing "CpWord regX[10]" to "CpWord *regX[10]"...

It is already a reference to a value, to which must be assigned (or stored to) another reference by way of An.  A pointer to the memory itself would remove the assignment operation on each Xn reference.  It'd be a PITA, but it would be a speedup of significant proportion, at least in theory, removing a sequence that may fit actual registers, but is still here just a memory/memory reference, and thus removing a redundant operation in how many executed instructions?

I am of the belief that this project shouldn't get like Hercules with it's pathetic performance.

**Comment by kej715** [comment]
Remember that X0 is not associated with A0 in the same that X1 - X7 are
associated with A1 - A7. Assigning a value to A0 has no effect on X0.

**Comment by kej715** [comment]
i.e., X0 does not have a direct relationship to memory. Also, instructions
such as BXi Xj+Xk do not affect memory.
